### ISSUE TITLE
[Bug] Platform not showing until at least 1 game is added

### ISSUE THREAD

**Issue by Subtixx** [rommapp/romm#1366]
**RomM version**
rommapp/romm:latest

**Describe the bug**
A platform that was just added does not show up in the "Scan" page filter.

**To Reproduce**

1. Add new Platform binding
2. Go to scan page
3. Try to filter by the newly added platform

**Expected behavior**
Find the platform in the dropdown

**Screenshots**
-

**Desktop (please complete the following information):**

- OS: Arch Linux
- Browser Firefox
- Version 133.0.3 (64-bit)

**Additional context**
-

**Comment by zurdi15** [comment]
Yeah we can maybe just show all the filesystem platforms, but for now you can use the "new platforms" option when scanning
